esphome/components/modbus/modbus.cpp

```diff
@@ -130,9 +178,11 @@ bool Modbus::parse_modbus_byte_(uint8_t byte) {
     uint16_t remote_crc = uint16_t(raw[data_offset + data_len]) | (uint16_t(raw[data_offset + data_len + 1]) << 8);
     if (computed_crc != remote_crc) {
       if (this->disable_crc_) {
-        ESP_LOGD(TAG, "Modbus CRC Check failed, but ignored! %02X!=%02X", computed_crc, remote_crc);
+        ESP_LOGD(TAG, "Modbus CRC Check failed, but ignored! %02X!=%02X  %s %dms after last send", computed_crc,
+                 remote_crc, format_hex_pretty(this->rx_buffer_).c_str(), millis() - this->last_send_);
```

Comment:
Split this up; leave dumping packet contents for (very_)verbose mode.
```suggestion
        ESP_LOGW(TAG, "CRC check failed; ignoring");
        ESP_LOGVV(TAG, "  (%02X != %02X)  %s %dms after last send", computed_crc,
                 remote_crc, format_hex_pretty(this->rx_buffer_).c_str(), millis() - this->last_send_);
```